core/tests/protractor_utils/forms.js

```diff
@@ -385,7 +385,10 @@ var RichTextChecker = function(arrayOfElems, arrayOfTexts, fullText) {
 
   var _readFormattedText = function(text, tagName) {
     expect(arrayOfElems[arrayPointer].getTagName()).toBe(tagName);
-    expect(arrayOfElems[arrayPointer].getInnerHtml()).toBe(text);
+    expect(
+      browser.executeScript(
```

Comment:
I went by the new Protractor Docs, was in one of their breaking changes [here](https://github.com/angular/protractor/blob/master/CHANGELOG.md#500). I can try the method you described above, seems much cleaner.